HISTORY.rst

```diff
@@ -45,7 +45,7 @@ History
 
 * Bug fix, error in manifest of files to include in python package (#120)
 
-Unreleased
+0.3.6 (2023-11-29)
 ----------------------
 
 * Updated datasets for MAAP STAC
```

Comment:
Hmm, didn't we remove Stamen base layers? This also impacts VEDA, and possibly all users if it's hard coded into the library.

Reply:
Temporarily disabled - just commented the layer out. The Stamen post said that they were migrating the maps to be hosted somewhere, so they might become available through ipyleaflet again. I'll update the log

Reply:
https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles/ it moved to Stadia which might incur costs to users and would require more work to implement a token authenticated layer.

My point was the Changelog should say remove/disabled Stamen layers.